_news/2021-07-23-welcome-no-borders-trans-safety-network.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: article
+author: Kim, Mallory & Iona
+title: No Borders Manchester & Trans Safety Network join Resistance Lab
+title_fancy: "No Borders Manchester &<br> <span>Trans Safety Network</span><br> join Resistance Lab"
+excerpt: By working with No Borders Manchester and Trans Safety Network we further expand our capacity to cover state violence and human rights injustices in Manchester and the UK. Both these groups bring unique perspectives of working with different structurally disadvantaged people and we look forward to producing formal outputs together.
+---
+
+Resistance Lab began as a simple provocation: can technology be used to help fight state violence, especially police brutality? 
+
+Beginning in 2018, representatives from Northern Police Monitoring Project and Geeks for Social Change facilitated a series of discussions between activists, campaigners, researchers, youth workers, academics and local residents. Over the first year our coalition grew to include both many individual members and group membership by Race, Roots and Resistance; The African Pot project; Kids of Colour; and Sites of Resistance.
+
+In August last year, we published our inaugural report [_A Growing Threat to Life: Taser usage by Greater Manchester Police_](/our-work/gmp-taser-update/index.html), which was then followed by a [report into the false claims made by GMP this year](/our-work/gmp-taser-update/index.html), and a [panel discussion on the use of Tasers nationally](/news/2021-02-25-no-more-tasers-panel.html). 
+
+Through this work we've been working closely with representatives from No Borders Manchester and Trans Safety Network who we are now delighted to announce are now formal group members of the Resistance Lab collective.xec
+
+## No Borders Manchester
+
+No Borders Manchester are an activist group which aims to challenge border regimes in the UK and across Europe. To do this we conduct a wide range of activities to bring people together in resistance. 
+
+These include campaigns to raise awareness and challenge the Home Office on issues like deportation and #Stoptheplane; English classes both in person and online; and the Signing Support Network which supports people going to 'sign' at Dallas Court Reporting centre. Resistance Lab was the impetus for [the imok bot we launched to support the Signing Support Network back in May](https://gfsc.studio/2021/05/12/imok-is-launched.html). No Borders Manchester also run a community football group, conversation classes, publish a No Borders zine, and launched the day-long No Borders Festival last year. 
+
+[https://nobordersmcr.com/](https://nobordersmcr.com/){:.btn} 
```

Comment:
The `{:.btn}` is making it render weirdly, with a trailing `>` on the end

Additionally, maybe remove the trailing slash

<img width="513" alt="Screenshot 2021-07-22 at 22 39 37" src="https://user-images.githubusercontent.com/84906/126712955-da5ffe27-5de7-4206-9176-de5ac39a965b.png">

Reply:
The > is deliberate and meant to make it look more button-y (its in use in other places in the site too)

Happy to remove them tho agree it does look a bit weird after a link